internal/printer/object_table.go

```diff
@@ -74,12 +81,31 @@ func (ol *ObjectTable) AddRowForObject(ctx context.Context, object runtime.Objec
 		return fmt.Errorf("get status for object: %w", err)
 	}
 
+	var pluginStatus *plugin.ObjectStatusResponse
+	var detailComponent *component.List
+	if ol.pluginManager != nil {
```

Comment:
The behaviour of objectTable with a plugin status can be tested for details and status updates.

Reply:
Done